framework/src/outputs/Output.C

```diff
@@ -114,6 +116,10 @@ Output::Output(const InputParameters & parameters)
                                            : -std::numeric_limits<Real>::max()),
     _end_time(isParamValid("end_time") ? getParam<Real>("end_time")
                                        : std::numeric_limits<Real>::max()),
+    _start_step(isParamValid("start_step") ? getParam<int>("start_step")
```

Comment:
I suggest using default values instead

Reply:
Good point. Does it always start at 0? If so, why is it not an unsigned int?

Reply:
See my comment above. Right now, step number is currently signed, but I don't think it needs to be. Let's leave it as signed for now and we'll roll through here later and change everything at once.